lib/ruby/truffle/rubysl/rubysl-timeout/lib/rubysl/timeout/timeout.rb

```diff
@@ -122,6 +122,7 @@ def self.watch_channel
 
   # Spin up the thread up front to avoid a thread collision problem spinning
   # it up lazily.
+  puts caller
```

Comment:
Thanks.  I had removed it, then needed it again, and forgot to remove it the second time :-/